src/parser/jobs/rpr/modules/Enshroud.tsx

```diff
@@ -13,6 +13,12 @@ import {DISPLAY_ORDER} from './DISPLAY_ORDER'
 
 const REAPINGS_PER_SHROUD = 4
 
+const SEVERITIES = {
+	1: SEVERITY.MINOR,
+	3: SEVERITY.MEDIUM,
```

Comment:
Nit; Given that all 4 reapings are greater 2 communios in potency in the same amount of time, I feel like the Severity could be harsher to the player but I'm okay with letting better reaper players than I fine-tune severities